kinetic/distribution.yaml

```diff
@@ -12614,6 +12614,16 @@ repositories:
       url: https://github.com/ros/ros_realtime.git
       version: hydro-devel
     status: unmaintained
+  ros_sensor_msgs:
```

Comment:
Since these repositories are not browseable, I cannot confirm, but this name does not look to meet our naming conventions. http://wiki.ros.org/ROS/Patterns/Conventions#Naming_ROS_Resources